controllers/articles.controllers.js

```diff
@@ -14,4 +14,14 @@ exports.getArticles = (req, res, next) => {
     selectArticles().then((data) => {
         res.status(200).send(data)
     })
+}
+
+exports.getArticleComments = (req, res, next) => {
+    const {article_id} = req.params
+    selectArticleComments(article_id).then((data) => {
+        res.status(200).send(data)
+    })
```

Comment:
We should be sending back the data on an appropriate key. I.e. `...send({comments: data})` this way when accessing this on the test/receiving side the key actually gives meaning to the data as well. 

This should be done for every endpoint as well